Sources/SwiftInspectorVisitors/Tests/ImportVisitorSpec.swift

```diff
@@ -80,12 +80,12 @@ final class ImportSyntaxReaderSpec: QuickSpec {
 
       }
 
-      context("with an  import statement with an attribute") {
+      context("with an import statement with an attribute") {
         beforeEach {
           let content = """
                         @_export import SomeModule
 
-                        public final class Some {}
+                        public final protocol Some {}
```

Comment:
did some sneaky changing of types in our tests in order to get coverage over the `.skipChildren` additions I made.

Might be better to write a performance test on this type that includes these types? Not sure how to do that with Nimble though.

Reply:
Yeah, I've been thinking for a while to write some performance test for this too.